src/main/java/uk/gov/companieshouse/company/links/tranformer/InsolvencyApiTransformer.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.company.links.tranformer;
```

Comment:
If we are keeping this package needs to be renamed to `transformer`

Reply:
Deleted the transformer class and this package as per the earlier comment.